lib/openapi_first.rb

```diff
@@ -68,6 +69,21 @@ def self.middleware(
     )
   end
 
+  def self.add_null_types!(schema_hash)
+    # end of recursion condition
+    return unless schema_hash.is_a?(Hash)
+
+    if schema_hash['nullable']
+      type = schema_hash.delete('type')
+      schema_hash['oneOf'] = [{ 'type' => 'null' }, { 'type' => type }] if type
```

Comment:
What will happen if the field is nullable, but does not have a type (which I'm not sure if is valid at all)?

E.g.
```yaml 
field:
  nullable: true
```

As far as I understand this snippet will just delete the `nullable: true`, correct?

Shouldn't it lead to something like?
```yaml 
field:
  oneOf:
    type: 'null'
    type: ???
```